guides/common/modules/con_web-ui-overview.adoc

```diff
@@ -4,7 +4,32 @@
 = {ProjectWebUI} overview
 
 You can manage and monitor your {Project} infrastructure from a browser with the {ProjectWebUI}.
-For example, you can use the following navigation features in the {ProjectWebUI}:
+ifdef::satellite[]
+{Project} supports recent versions of the Mozilla Firefox and Google Chrome browsers.
+endif::[]
+ifdef::foreman-el,foreman-deb,katello[]
+{Team} recommends the following major browsers:
+
+* Apple Safari
+* Google Chrome
+* Microsoft Edge
+* Mozilla Firefox
+* Mozilla Firefox Extended Support Release (ESR)
+
+[NOTE]
+====
+Use the latest versions of a major browser.
+Other browsers and their older versions might work unpredictably.
+====
+endif::[]
+
+ifdef::satellite[]
+The {ProjectWebUI} and command-line interface support English, French, Japanese, Korean, and Simplified Chinese.
+endif::[]
+ifndef::satellite[]
+The {ProjectWebUI} and command-line interface is translated into various languages.
+endif::[]
```

Comment:
We're mixing web UI and CLI info here. Perhaps it would be worth splitting?

Reply:
That's a good point. However, where would the command-line part go? And does it include only Hammer CLI or also the other commands like the installer?

I think that considering that this is currently included in the [browser section of the installation guide](https://docs.theforeman.org/3.15/Installing_Server/index-katello.html#supported-browsers_foreman), we shouldn't consider it a blocker for this PR. However, if you have a specific suggestion that I could implement quickly, I don't mind adding it to this PR :)